doc/rst/developer/chips/20.rst

```diff
@@ -100,8 +100,8 @@ What's going wrong? The isMoreVisible rule in function resolution uses
 point-of-instantiation to determine visibility.
 
 Solution: remove point-of-instantiation from consideration in
-determination of more visible function in disambiguation. A branch
-exploring this idea passes testing.
+determination of more visible function in disambiguation. This behavior
+is no longer present on master today.
```

Comment:
I'd say something more specific, such as "This solution has been adopted as of Chapel 1.18."